user_news/models.py

```diff
@@ -74,6 +74,7 @@ class Outage(News):
     end_date = models.DateTimeField('expected end of outage')
     resolved = models.BooleanField('resolved', default=False)
     send_email_notification = False
+    send_email_reminder = False
```

Comment:
We will also need a DB migration for this. I don't remember the exact invocation to do this, but I think Django makes it pretty easy to auto-generate them!

Reply:
Based on my previous comment above, it's unclear if we need these here. We do not store send_email_notification or send_email_reminder boolean values in the database so no migration is needed.